nilearn/region_extraction/SimpleThresholdExtractor.py

```diff
@@ -0,0 +1,139 @@
+
```

Comment:
You don't need an empty line here. Plus, you should not use `CamelCase` for file names. In python, we use `snake_case`.